ServerCore/ServerCore.csproj

```diff
@@ -24,7 +24,12 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.All" Version="2.1.2" />
     <PackageReference Include="Microsoft.AspNetCore.Authentication.MicrosoftAccount" Version="2.1.2" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.2.2" />
```

Comment:
I believe these packages get automatically included by other packages, so we shouldn't need them explicitly added or upgraded. Did you hit issues without them?